gammapy/maps/region/geom.py

```diff
@@ -417,35 +418,37 @@ def to_binsz_wcs(self, binsz):
 
         Parameters
         ----------
-        binzs : float, string or `~astropy.quantity.Quantity`
+        binsz : float, string or `~astropy.quantity.Quantity`
```

Comment:
```suggestion
        binsz : float or string or `~astropy.quantity.Quantity`
```

Reply:
When you have a list, the 'or' is only written between the last two quantities.